weave/trace/weave_client.py

```diff
@@ -1860,6 +1860,15 @@ def _op_runs(self, op_def: Op) -> Sequence[Call]:
     def _ref_uri(self, name: str, version: str, path: str) -> str:
         return ObjectRef(self.entity, self.project, name, version).uri()
 
+    def flush(self) -> None:
+        """
+        An optional flushing method for the client.
+        Forces all background tasks to be processed, which ensures parallel processing
```

Comment:
I think you should add an entry in faq section or troubleshooting docs for evals